src/components/UploadVideo/UploadVideo.tsx

```diff
@@ -65,6 +59,7 @@ const UploadVideo = () => {
   const handleFileInputChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
     if (event.target.files) {
       source.current = event.target.files[0];
+      videoName.current = event.target.files[0].name.replace(/\.\w+$/gi, ''); //TODO send videoName.current to backend and display it in the dashboard
     }
```

Comment:
splitting video name from the extension?

Reply:
Reading my code as an open book!